README.md

```diff
@@ -24,14 +24,18 @@ Install the package with:
 The package needs to be configured with your account's API key which you can get by opening a ticket in our Discord for now. Our developer dashboard is coming soon...
 
 ```js
-import { MarketPlace, Inscription, Satscanner, Satextractor, Mempool } from "ordinalsbot";
+
+import { Ordinalsbot } from 'ordinalsbot'
 
 // if no parameter given, default environment is 'live'
-let inscription = new Inscription("API_KEY", "dev");
-let marketplace = new MarketPlace("API_KEY", "dev");
-let satscanner = new Satscanner("API_KEY", "dev");
-let satextractor = new Satextractor("API_KEY", "dev");
-let mempool = new Mempool("API_KEY", "dev");
+const ordinalsbotObj = new Ordinalsbot(API_KEY, 'dev')
+const marketPlace = ordinalsbotObj.MarketPlace()
+const inscription = ordinalsbotObj.Inscription()
+const launchpad = ordinalsbotObj.Launchpad()
+const mempool = ordinalsbotObj.Mempool()
+const satextractor = ordinalsbotObj.Satextractor()
+const satscanner = ordinalsbotObj.Satscanner()
```

Comment:
will this change break currently running apps? 
if so, we need to bump a major version right?

Reply:
Yeah, I need to sort out the formatting

Reply:
so what's the solution? will you change it to make it backwards compatible or are you bumping a major version?